spock-specs/src/test/groovy/org/spockframework/mock/runtime/ByteBuddyMockFactoryConcurrentSpec.groovy

```diff
@@ -39,7 +40,7 @@ class ByteBuddyMockFactoryConcurrentSpec extends Specification {
   }
 
   // Just to be save to abort, normally the tests run in 2 secs.
-  @Timeout(40)
+  @Timeout(120)
```

Comment:
❓ Why this 3x increase?
40s is already really long compared to all the other tests, I'd like to keep running Spock tests as fast as reasonably possible.

Do we need to separate out these probabilistic tests to some scheduled ci jobs?

Reply:
The tests are flaky under load on my machine, so I increased the timeout, but this is not a real fix.
Maybe we would need to make them `@Isolated` or separate them as you proposed.

If I run them without any other tests, they are fast, but with congestion, they tend to break the timeout.